rpcutil/rpcutil.go

```diff
@@ -153,6 +158,16 @@ func (cr llCodecRequest) ReadRequest(args interface{}) error {
 		}
 	}
 
+	if cr.c.RunInputApplicators {
+		if err := applicator.Apply(args); err != nil {
+			llog.Error("error running applicator on args", cr.kv, llog.KV{"err": err})
```

Comment:
If an applicator errors then it probably isn't a problem with the input and there's something wrong with the tag. I don't think the applicators should really be validating the input at all and just erroring on unexpected types essentially.

i.e. applied `trim` to an `int` or something like that.